pkg/workflows/wasm/pb/wasm.proto

```diff
@@ -24,6 +24,7 @@ message Request {
     ComputeRequest computeRequest = 3;
     google.protobuf.Empty specRequest = 4;
   }
+  string triggerId = 5;
 }
 
 message ComputeResponse { capabilities.CapabilityResponse response = 1; }
```

Comment:
I think you can get rid of the protos down below under the comment:
`// TODO: v2 structs to be moved to a different package`

You added equivalents inside v2 directories and converted Module V2 interface.